quickwit/quickwit-control-plane/src/model/shard_table.rs

```diff
@@ -111,6 +111,31 @@ impl ShardTableEntry {
     }
 }
 
+#[derive(Default)]
```

Comment:
It's confusing that we are tracking the locations of shards on ingesters, i.e. where the data is ingested vs. indexers i.e. where the data is indexed, yet use the variables and names `indexer_id` or `indexer`.

Reply:
it is tricky because ingesters are always indexers today, and in this code in particulary, we are trying to apply logic where we assign on indexers, depending on whether they host the shard or not in their "indexer role". 

We can rename stuff, we will still in some place pass a indexer_id to something calling the variable `ingester_id`.

I'll rename it here if you think that helps.